atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingKeyValueService.java

```diff
@@ -266,8 +273,8 @@ public void dropTables(Set<TableReference> tableRefs) {
     public ClosableIterator<List<CandidateCellForSweeping>> getCandidateCellsForSweeping(TableReference tableRef,
             CandidateCellForSweepingRequest request) {
         //noinspection unused - try-with-resources closes trace
-        try (CloseableTrace trace = startLocalTrace("getCandidateCellsForSweeping({}, ts {})",
-                tableRef, request.maxTimestampExclusive())) {
+        try (CloseableTrace trace = startLocalTrace("getCandidateCellsForSweeping({}, ts {}).next()",
```

Comment:
i think we should just remove the tracing on these iterator methods, and maybe just comment that it doesn't make sense. it's misleading to have it